spec/factories/users.rb

```diff
@@ -1,18 +1,10 @@
 FactoryGirl.define do
   factory :user do
+    id "1"
```

Comment:
don't do this. 

"manually" setting "id" values is a _serious_ anti pattern. If you are curious I'd highly suggest googling there are quite a few blog posts/etc written about this.